src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -645,9 +645,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.BMP;
 		} else if (value.equals("tiff")) {
 			format = FormatConfiguration.TIFF;
+		// Value can look like "Advanced@L1", "Complex@L2", "MP@LL" or "Simple@L3" with VC-1 or MPEG-4 Visual at this point.
 		} else if (containsIgnoreCase(value, "@l") && streamType == StreamType.Video) {
-			media.setAvcLevel(getAvcLevel(value));
-			media.setH264Profile(getAvcProfile(value));
+		//TODO: Is this really needed, it's already set earlier..?
```

Comment:
> Since no format filter is applied, as i did, and that the profile is only used for the database, AFAIK, using only that "@" will log again more video profiles.

Sorry, I don't understand.

> What is really the purpose of the video profile ?

The video profile is used to parse H264 and H265 level and profile for now. I imagine it can be used for other things as well in the future.